src/components/AllServices/AllServicesSection.tsx

```diff
@@ -4,32 +4,32 @@ import { ITLess } from '../../utils/common';
 import AllServicesGroup from './AllServicesGroup';
 import AllServicesIcons from './AllServicesIcons';
 import AllServicesLink from './AllServicesLink';
-import type { AllServicesGroup as AllServicesGroupType, AllServicesLink as AllServicesLinkType, AllServicesSection } from './allServicesLinks';
+import { AllServicesLink as AllServicesLinkType, AllServicesSection, isAllServicesGroup, isAllServicesLink } from './allServicesLinks';
 
 export type AllServicesSectionProps = AllServicesSection;
 
-export const isAllServicesGroup = (item: AllServicesGroupType | AllServicesLinkType): item is AllServicesGroup => {
-  return (item as AllServicesGroupType).isGroup === true;
-};
-
 const AllServicesSection = ({ icon, title, description, links }: AllServicesSectionProps) => {
-  const TitleIcon = AllServicesIcons[icon];
-  const filteredLinks = ITLess() ? links.filter((link) => link.ITLess) : links;
+  const TitleIcon = AllServicesIcons[icon as keyof typeof AllServicesIcons];
+  const filteredLinks = ITLess() ? links.filter((link) => (link as AllServicesLinkType).ITLess) : links;
   return (
     <Card className="pf-u-display-block pf-u-mb-md pf-u-background-color-100">
       <CardTitle>
         <Icon className="pf-u-mr-xs" isInline>
-          <TitleIcon />
+          {TitleIcon && <TitleIcon />}
         </Icon>
         {title}
       </CardTitle>
       <CardBody>
         <TextContent className="pf-u-font-size-sm">
           <Text component={TextVariants.p} className="pf-u-mb-md">
-            {description}
+            {description || null}
           </Text>
           {filteredLinks.map((link, index) =>
-            isAllServicesGroup(link) ? <AllServicesGroup key={index} {...link} /> : <AllServicesLink key={index} {...link} />
+            isAllServicesGroup(link) ? (
+              <AllServicesGroup key={index} {...link} />
+            ) : (
+              isAllServicesLink(link) && <AllServicesLink key={index} {...link} />
```

Comment:
You should not require the else branch type guard. Now that we don't typecast the first guard argument, TS will correctly evaluate the else branch type.

Reply:
Ah, good point! I had to use it because we had string in the services link, but that's no longer the case. No extra guard is required